dandi/cli/cmd_upload.py

```diff
@@ -39,7 +39,10 @@
 )
 @click.option(
     "--validation",
-    help="Data must pass validation before the upload.  Use of this option is highly discouraged.",
+    help="Controls validation requirements before upload. "
```

Comment:
Let's keep the message!

```suggestion
    help="Controls validation requirements before upload. Use of this option is highly discouraged. "
```

Reply:
@yarikoptic I applied my own suggested change mentioned above. Please re-review.